modules/apps/27-interchain-accounts/host/keeper/account_test.go

```diff
@@ -0,0 +1,33 @@
+package keeper_test
```

Comment:
Is this from a merge conflict?

Reply:
do you mean the entire test file?

Reply:
Yea. Why was the test file added for a version bump pr?

Reply:
Looks like I cut this branch before the PR which removed this. So when I pulled in the latest changes this file was kept.

I'll push an update to remove now. Good catch! :)